sdk-artifact-sync.py

```diff
@@ -123,18 +134,18 @@
 if verbose:
     print
 
-
```

Comment:
Tsk tsk this violates PEP 8! I see you're not using PyCharm for editing this file or it would yell at you!